x-pack/plugins/security_solution/common/endpoint/schema/actions.test.ts

```diff
@@ -185,20 +185,25 @@ describe('actions schemas', () => {
       }).not.toThrow();
     });
 
-    it.each(['isolate', 'unisolate', 'kill-process', 'suspend-process', 'running-processes'])(
-      'should work with commands query params with %s action',
-      (command) => {
-        expect(() => {
-          EndpointActionListRequestSchema.query.validate({
-            page: 10,
-            pageSize: 100,
-            startDate: new Date(new Date().setDate(new Date().getDate() - 1)).toISOString(), // yesterday
-            endDate: new Date().toISOString(), // today
-            commands: command,
-          });
-        }).not.toThrow();
-      }
-    );
+    it.each([
+      'isolate',
```

Comment:
We have a `const` with all of the command names here:

https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/endpoint/service/response_actions/constants.ts#L10-L21

could we use that instead of hard coding the list here?

Reply:
done 097ac5b410e1a531b86a826e74fb884f3dfe2929